src/main/java/faang/school/godbless/streamapi_1/StreamLib.java

```diff
@@ -0,0 +1,44 @@
+package faang.school.godbless.streamapi_1;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+public class StreamLib {
+    public static List<Integer> evenList(List<Integer> list) {
+        return list.stream().filter(num -> num % 2 == 0).toList();
+    }
+
+    public static Integer maxNumber(List<Integer> list) {
+        return list.stream().mapToInt(Integer::intValue).max().orElse(0);
+    }
+
+    public static Double averageList(List<Integer> list) {
+        return list.stream().mapToInt(Integer::intValue).average().orElse(0);
+    }
+
+    public static Integer grepCount(List<String> list, String pattern) {
+        return (int) list.stream().filter(s -> s.startsWith("a")).count();
```

Comment:
Why "a", if you have "pattern" as a parameter?

Reply:
Oooops :)